packages/core/src/state-management/running-state.ts

```diff
@@ -8,31 +8,16 @@ import {
 import { StreamID } from '@ceramicnetwork/streamid'
 import CID from 'cids'
 
-/**
- * Describes the source from which this RunningState was initialized with its StreamState
- */
-export enum StateSource {
-  /**
-   * Indicates that the stream was pinned and its state was loaded from the StateStore. This will cause runningState to keep track of the stored commit CIDs.
-   * The stored commit CIDs can be used to prevent commits from being stored again.
-   */
-  STATESTORE,
-  /**
-   * Indicates that the Stream was loaded from the network.
-   * runningState does not keep track of the commit CID's of the state as they may not have been stored.
-   */
-  NETWORK,
-}
 export class RunningState extends StreamStateSubject implements RunningStateLike {
   readonly id: StreamID
   readonly subscriptionSet: SubscriptionSet = new SubscriptionSet()
-  pinnedCommits?: Set<string>
+  private _pinnedCommits?: Set<string>
```

Comment:
made this private and added a getter so that external callers can't reassign it without going through a method on RunningState.  FYI @stephhuynh18